### PR TITLE
Two panel responsive collapsing columns pass one

### PR DESCRIPTION
**Pull request by capajon**
@dlockhart This is related to the "useResponsiveCollapseLayout" changes in the LE that we were holding off putting in 10.6.6.

### CODE REVIEW

File: src/page-two-panel-selector.scss [hunk 1]
```diff
@@ -58,3 +58,19 @@
 .d2l-twopanelselector-padding {
 	padding-top: 30px;
 }
+
+@media(max-width: 1000px) {
+	.d2l-twopanelselector-wrapper.d2l-repsonsive-collapse-layout {
+		.d2l-box-layout {
```
Comment (omsmith) [inline src/page-two-panel-selector.scss]: All of these should likely be `> ...`

Comment (capajon) [inline src/page-two-panel-selector.scss]: Right didn't notice that - seems critical for box-layout.  I don't think it should matter on the others but I can put it there too for safety. Will update.

**Comment by omsmith** [comment]
Can't comment on the correctness, but looks safer

**Comment by dlockhart** [comment]
Tried this out, seems to work well! LGTM

**Pull request merged**